<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>437</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>426</width>
    <height>437</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>死亡愛麗絲 競技場助手</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QTextEdit" name="background">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>821</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-2</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>夢魘換位計算</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>設定</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>78</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>預留換位時間：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="reserveTime">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>78</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>78</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>競技場時段：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="JJC_Time">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>35</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <item>
       <property name="text">
        <string>08：00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12：30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18：00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19：00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20：00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21：00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22：00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23：00</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>123</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>判斷技能時間：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="determineTime">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>123</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>123</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>換位時間：</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>280</y>
       <width>501</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTimeEdit" name="switchTime">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>302</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>60</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="maximumDate">
      <date>
       <year>2020</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2020</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::MinuteSection</enum>
     </property>
     <property name="displayFormat">
      <string>mm : ss</string>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>20</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>現在時間</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QTimeEdit" name="currentTime">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>37</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximumDate">
       <date>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>hh : mm : ss</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>171</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>競技場剩餘時間</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QTimeEdit" name="remainingTime">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>37</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="date">
       <date>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="maximumDate">
       <date>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MinuteSection</enum>
      </property>
      <property name="displayFormat">
       <string>mm : ss</string>
      </property>
      <property name="time">
       <time>
        <hour>12</hour>
        <minute>20</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>171</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>DeBuff</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="debuff_single_lv4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;準備時間：60秒&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;*尚未推出相關技能的夢魘&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Ⅳ</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="debuff_single_lv3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;準備時間：70秒&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;img src=&quot;:/images/debuff_single_lv3_1.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_single_lv3_2.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_single_lv3_3.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_single_lv3_4.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Ⅲ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>降單攻 / 防</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>降雙攻 / 防</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="debuff_both_lv4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;準備時間：40秒&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;img src=&quot;:/images/debuff_both_lv4_1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Ⅳ</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="debuff_both_lv3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;準備時間：50秒&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;img src=&quot;:/images/debuff_both_lv3_1.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_2.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_3.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_4.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_5.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_6.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_7.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_8.png&quot;/&gt;&lt;img src=&quot;:/images/debuff_both_lv3_9.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Ⅲ</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>其他</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>426</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>嗯...還沒想到要寫什麼...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
